src/main/java/stream/flarebot/flarebot/util/ReportManager.java

```diff
@@ -0,0 +1,62 @@
+package stream.flarebot.flarebot.util;
+
+import org.joda.time.DateTime;
+import stream.flarebot.flarebot.objects.Report;
+import stream.flarebot.flarebot.objects.ReportStatus;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReportManager {
+
```

Comment:
You refering to make the class private and final or the varibles? because you can't make a class final